torch/utils/cpp_extension.py

```diff
@@ -719,6 +762,32 @@ def unix_wrap_ninja_compile(sources,
                 cuda_dlink_post_cflags = unix_cuda_flags(extra_postargs['nvcc_dlink'])
             else:
                 cuda_dlink_post_cflags = None
+
+            sycl_post_cflags = None
+            sycl_cflags = None
+            sycl_dlink_post_cflags = None
+            if with_sycl:
+                sycl_cflags = extra_cc_cflags + common_cflags + COMMON_SYCL_FLAGS
+                if isinstance(extra_postargs, dict):
+                    sycl_post_cflags = extra_postargs['sycl']
+                else:
+                    sycl_post_cflags = list(extra_postargs)
+                sycl_cflags += COMMON_SYCL_CCFLAGS
+                append_std17_if_no_std_present(sycl_cflags)
+                _append_sycl_std_if_no_std_present(sycl_cflags)
+                host_cflags = extra_cc_cflags + common_cflags + post_cflags
+                append_std17_if_no_std_present(host_cflags)
+                # escaping quoted arguments to pass them thru DPC++ compiler
```

Comment:
Sounds like confused. Here we introduced DPC++ compiler brand...
Should we unify the name in both comments and code line?
`dpc++` or `sycl compiler`

Reply:
ok, I switched to `SYCL compiler` in the comments to be consistent with the code.